Source/WebCore/Modules/mediastream/MediaStreamTrack.cpp

```diff
@@ -618,6 +618,11 @@ void MediaStreamTrack::trackSettingsChanged(MediaStreamTrackPrivate&)
     configureTrackRendering();
 }
 
+void MediaStreamTrack::trackConfigurationChanged(MediaStreamTrackPrivate&)
+{
+    queueTaskToDispatchEvent(*this, TaskSource::Networking, Event::create(eventNames().configurationchangeEvent, Event::CanBubble::No, Event::IsCancelable::No));
```

Comment:
The [spec says](https://w3c.github.io/mediacapture-extensions/#exposing-change-of-mediastreamtrack-configuration) not to fire the event immediately if `track.muted` is true.

Reply:
Ah right, I added the check in GPUProcess but it is better to check it here again.

Reply:
Yep, I will remove it.

Reply:
The remove it was for the next comment.
I added a muted/ended check